build.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
-VERSION="1.1.5"
-MILESTONE="master"
-RPM_RELEASE="1.1.$MILESTONE.$(date -u +%Y%m%d%H%M%S)"
+VERSION="1.2.0"
```

Comment:
why 1.2.0 and not 1.1.5?

Reply:
I believe supporting python-3 gives us a new functionality, so in this case, we should bump the minor version

Reply:
Sure, that's up to you what versioning you decide to do. But in all other roles we are still stick to 1.1.x, so it may be confusing for someone.